src/azure-cli/setup.py

```diff
@@ -106,7 +106,7 @@
     'azure-mgmt-monitor~=5.0.0',
     'azure-mgmt-msi~=7.0.0',
     'azure-mgmt-netapp~=10.0.0',
-    'azure-mgmt-policyinsights~=1.1.0b2',
+    'azure-mgmt-policyinsights~=1.1.0b4',
```

Comment:
For the beta version, we usually recommend fixing it to avoid unexpected issues caused by breaking changes from later version in the future
```suggestion
    'azure-mgmt-policyinsights==1.1.0b4',
```